app/templates/dashboard.html

```diff
@@ -69,7 +69,20 @@ <h1 class="govuk-heading-xl">Your applications</h1>
     'rows': ns.rows
     }) }}
 {% endif %}
-<form method="POST" action={{ url_for('routes.new')}}>
+
+{% if applications|length > 0 %}
+<form method="POST" action={{url_for('routes.new')}}>
+    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+    <input type="hidden" name="account_id" value="{{ account_id }}">
+    <input type="hidden" name="fund_id" value="{{ fund_id }}">
+    <input type="hidden" name="round_id" value="{{ round_id }}">
+    {{ govukButton({
+    'text': "Start new application",
+    'classes': "govuk-button--secondary"
```

Comment:
Ah I think it may be possible to add a conditional after "govuk-button--secondary" like `"govuk-button--secondary" if applications|length > 0` - does it work? If that's the only requirement.

Reply:
It didn't work for me 🤔.  Also was unable to call the govuk css class  to the macro so had to create macro for both buttons.